packages/components/src/MultiSelect/MultiSelect.tsx

```diff
@@ -30,9 +35,12 @@ export const MultiSelect = ({
   isOpen,
   onOpenChange,
   classNameOverride,
+  status,
+  validationMessage,
   ...restProps
 }: MultiSelectProps): JSX.Element => {
   const id = propsId ?? useId()
+  const validationId = `${id}-description`
```

Comment:
You'll be needing to combine with the description, so they need different ids :)
```suggestion
  const validationId = `${id}--validation-message`
```